### ISSUE TITLE
Love Swashbuckle, trying out Asp.Net Core, made a demo project

### ISSUE THREAD

**Issue by sgwatgit** [domaindrivendev/Ahoy#102]
Hey,

I'm a huge fan of the Swashbuckle library and I use it on all of my Web Api 2 projects. I've been following along with the development of the Core frameworks and was happy to see Swashbuckle is already being converted over.

I made a quick demo project to make sure I could wire everything up correctly https://github.com/sgwatgit/CoreWebApi

I plan to explore more Web Api and Ahoy functionality in it - not sure if it's worth linking to as an example at the moment but I might build it up enough for it to become interesting.

Basically just stopping by to say hello and thanks.

**Comment by domaindrivendev** [comment]
Hey - thanks so much for your support, it's so encouraging to get messages like this. Because feedback typically comes through an "issue" list the tone can, at times, be a little negative - they are reporting issues after all. So, this has made my Friday. Thanks!

Keep me posted on your sample project and I'd be happy to point to it for examples

**Comment by cwe1ss** [comment]
+1 on this. this is a great project :)
